testapi.pm

```diff
@@ -932,7 +932,7 @@ sub assert_script_run {
             # not change default timeout.
             timeout      => 90,
             fail_message => '',
-            quiet        => undef
+            quiet        => testapi::get_var('_QUIET_SCRIPT_CALLS')
```

Comment:
I don't think it's a good idea to access the test API within the methods itself. Do you think it would work to just access the `$bmwqemu::VARS` hash here?

Reply:
Why not?

Reply:
because the external API is the contract we give to the customers + error checking, etc. The internal calls we can decide within the context of os-autoinst how to use and skip some indirection.